mautic/mautic
# added css to center contact name in search result

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | staging for features or enhancements / 3.2 for bug fixes 
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #4758 




#### Description:



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. If you don't have a contact in your your DB, add one
3. Go to the search bar
![image](https://user-images.githubusercontent.com/31535432/100094275-0d988b00-2e59-11eb-879b-144dfb1ae48e.png)
4. Search for your contact
5. See that contact names are center align
![image](https://user-images.githubusercontent.com/31535432/100094386-3d479300-2e59-11eb-8e3b-9604e559d609.png)
